src/main/connector/database/Database/Database.yaml

```diff
@@ -0,0 +1,217 @@
+metrics:
+  db.aborted.clients:
+    description: The number of connections that were aborted.
+    type: Counter
+    unit: "{connection}"
+  db.aborted.connects:
+    description: The number of failed attempts to connect to the MySQL server.
+    type: Counter
+    unit: "{connection}"
+  db.io.size{db.io.direction="received"}:
+    description: The number of bytes received from all clients.
+    type: Counter
+    unit: By
+  db.io.size{db.io.direction="sent"}:
+    escription: The number of bytes sent to all clients.
+    type: Counter
+    unit: By
+  db.connections:
+    description: The number of connection attempts (successful or not) to the MySQL server.
+    type: Counter
+    unit: "{connection}"
+  db.client.connection.count{db.client.connection.state="used"}:
```

Comment:
The metric name shouldn't contain the value of the attribute. E.g. db.client.connection.count should be defined as the following:
```yaml
 db.client.connection.count:
    description: The number client connections to the database.
    type: UpDownCounter
    unit: "{connection}"
```
For instance, the `db.buffer.pool.operations{db.operation.direction="read|write"}` should be defined as the following (In MySQL.yaml):
```yaml
  db.buffer.pool.operations:
    description: The number of logical read or write requests made to the InnoDB buffer pool.
    type: Counter
    unit: "{request}"
```